examples/202-NN_examples.ipynb

```diff
@@ -42,7 +42,7 @@
   },
```

Comment:
<p>Looks weird, why do we have SMAPE about 5-6 with PytorchForecastingDatasetBuilder</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/268/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>You didn't change anything, why did the metric change?</p>